pom.xml

```diff
@@ -1457,6 +1468,9 @@
         </profile>
         <profile>
             <id>ci</id>
+            <properties>
+                <jacoco.skip>false</jacoco.skip>
+            </properties>
```

Comment:
I'm pulling this up locally, to poke around.  I'll get back to you shortly.

Thanks again for keeping this moving!!

Reply:
WOW, I didn't realize how slow the jacoco merge was. I haven't seen this with other builds